frontend/components/navbar.js

```diff
@@ -3,8 +3,10 @@ import Ratingbutton from "@/components/ratingbutton";
 import Ratebox from "@/components/ratebox";
 import Registrationbox from "@/components/registrationbox";
 import { useState } from "react";
+import { useSession, signOut } from "next-auth/react";
 
 export default function navbar() {
+  const { status } = useSession();
```

Comment:
You can now use the `useSession()` function inside every frontend file. It returns the current user authentication state. You can use it to conditionally show elements (e.g. logout button as in the example below).